### DISCUSSION TITLE
URL for FERMATA

### DISCUSSION THREAD

**Discussion by Mirekbfive** [iptv-org/community#331] [Q&A] [answered]
Žádám někoho, kdo by převedl nebo upravil url pro aplikaci FERMATA. Nevím kde se zeptat a kdo by mi pomohl. Jedná se například o tyto programy

I am asking for someone to convert or modify the url for the FERMATA app. I don't know where to ask and who would help me. These are, for example, these programs

http://rtmp.elektrika.cz/live/myStream.sdp/playlist.m3u8
#EXTINF:-1 tvg-id="ExtasyTVHD.cz" tvg-name="Extasy TV HD" tvg-country="CZ" tvg-language="Czech" tvg-logo="" group-title="XXX" ,Extasy TV HD
http://213.151.233.20:8000/dna-6233-tv-pc/hls/4004v105.m3u8
#EXTINF:-1 tvg-id="Nova Action-CZ" tvg-name="Nova Action-CZ" tvg-country="CZ" tvg-language="Czech" tvg-logo="https://nahrát. wikimedia.org/wikipedia/commons/d/d2/Nova-action-logo-2017.png" group-title="",Nova Action
http://92.62.234.223/102/mystream.m3u8
#EXTINF:-1 tvg-id="Nova Action.CZ" tvg-name="Nova Action-CZ" tvg-country="CZ" tvg-language="Czech" tvg-logo="https://nahrát. wikimedia.org/wikipedia/commons/d/d2/Nova-action-logo-2017.png" group-title="",Nova Action
https://nova-live.ssl.cdn.cra.cz/channels/nova_action_avod/playlist/cze/live_hq.m3u8
#EXTINF:-1 tvg-id="Nova Cinema.CZ" tvg-name="Nova Cinema-CZ" tvg-country="CZ" tvg-language="Czech" tvg-logo="https://static. epg.best/cz/NovaCinema.cz.png" group-title="",Nova Cinema
https://nova-live.ssl.cdn.cra.cz/channels/nova_cinema_avod/playlist/cze/live_hq.m3u8
#EXTINF:-1 tvg-id="Nova Gold.CZ" tvg-name="Nova Gold-CZ" tvg-country="CZ" tvg-language="Czech" tvg-logo="https://static. epg.best/cz/NovaGold.cz.png" group-title="",Nova Gold
https://nova-live.ssl.cdn.cra.cz/channels/nova_gold_avod/playlist/cze/live_hq.m3u8
#EXTINF:-1 tvg-id="Nova.CZ" tvg-name="Nova" tvg-country="CZ" tvg-language="čeština" tvg-logo="https://static.epg.best/ cz/Nova.cz.png" group-title="",Nova
https://nova-live.ssl.cdn.cra.cz/channels/nova_avod/playlist/cze/live_hq.m3u8
#EXTINF:-1 tvg-id="Nova2.CZ" tvg-name="Nova 2" tvg-country="CZ" tvg-language="Czech" tvg-logo="https://static.epg.best /cz/Nova2.cz.png" group-title="",Nova 2

**Comment by BellezaEmporium** [comment]
I don't quite understand what's the question here.

**Comment by Mirekbfive** [comment]
Hello.  Please a lot 🙏  I would need a URL for TV stations in Czech.  And
I can't find TV stations.  I need them for android car, for the fermata
application.

Dne so 25. 3. 2023 13:34 uživatel Ismaël Moret ***@***.***>
napsal:

**Comment by Mirekbfive** [comment]
I couldn't play stations 1 to 17 in fermata and I don't know why

https://github.com/iptv-org/iptv/pull/2362/commits/36a9f949fa72709bc084394e9d63bc843aad6213

Dne so 25. 3. 2023 13:45 uživatel Miroslav Folk ***@***.***>
napsal:

**Reply by BellezaEmporium** [reply]
Ah you're using a very old version of this file too.

The master link is here : https://raw.githubusercontent.com/iptv-org/iptv/master/streams/cz.m3u

**Comment by Mirekbfive** [comment]
And when I insert this link, everything works without any problems


https://raw.githubusercontent.com/Free-TV/IPTV/master/playlist.m3u8

Dne so 25. 3. 2023 13:49 uživatel Miroslav Folk ***@***.***>
napsal:

**Reply by BellezaEmporium** [reply]
I guess it's linked to how the channels are seen by your program (if it reads the tvg-name instead of the name after)

**Comment by Mirekbfive** [comment]
I don't understand this friend at all. I just pasted the url with the tv
shows. But the programs I want don't go there and I don't know how to
modify them to work

so 25. 3. 2023 v 13:57 odesílatel Ismaël Moret ***@***.***>
napsal:

**Reply by BellezaEmporium** [reply]
We do have a specific way of naming streams as we only have the tvg-id and the rest is generated through the database.

The closest type of file we have to what you've linked us could be this : https://iptv-org.github.io/iptv/countries/cz.m3u

**Comment by Mirekbfive** [comment]
Can you help me somehow? convert in some program?

so 25. 3. 2023 v 14:01 odesílatel Miroslav Folk ***@***.***>
napsal:

**Comment by Mirekbfive** [comment]
yes this room works without problems but i don't have the programs i sent
you...like nova, nova cinema, nova gold etc.

so 25. 3. 2023 v 14:04 odesílatel Ismaël Moret ***@***.***>
napsal:

**Reply by BellezaEmporium** [reply]
You can still ask for those channels in the Issues tab. Be sure to do one Issue per channel.

**Comment by Mirekbfive** [comment]
I still don't understand you. As I say, I'm a layman and I don't understand
any of this. I would need to send them these channels in jpg and put them
in a file like this https://iptv-org.github.io/iptv/countries/cz.m3u to run
it.

so 25. 3. 2023 v 14:16 odesílatel Ismaël Moret ***@***.***>
napsal:

**Reply by BellezaEmporium** [reply, marked as answer]
The channels you are asking for aren't in our playlists. You can propose them within our Issues tab available above.

There's a "New issue" button in which you can, afterwards, create a Channel Request form.

You can do it for each channel that's missing inside our playlists. But please check if there's an opened issue before creating a new one.

**Comment by Mirekbfive** [comment]
or I already found this link, but I need to put it in some file so that it
works in fermata


https://www.m3u8.cz/showthread.php?tid=967

so 25. 3. 2023 v 14:32 odesílatel Miroslav Folk ***@***.***>
napsal:

**Comment by Mirekbfive** [comment]
Hello, I am sending another file.  Which when I insert it, it cannot be
played.  You can look at it?



Dne so 25. 3. 2023 13:34 uživatel Ismaël Moret ***@***.***>
napsal:

**Comment by Mirekbfive** [comment]
Hello Friend. can you tell me how to insert a new thread? I can't create
anything in the channel problem.Thank you

so 25. 3. 2023 v 22:17 odesílatel Ismaël Moret ***@***.***>
napsal:

**Reply by BellezaEmporium** [reply]
https://github.com/iptv-org/iptv/issues/new?assignees=&labels=channel+request&template=------channel-request.yml&title=Add%3A+

One for each.
